src/run_perf_test.py

```diff
@@ -11,23 +11,6 @@
 from test_workflow.perf_test.perf_test_cluster import PerfTestCluster
 from test_workflow.perf_test.perf_test_suite import PerfTestSuite
 
-"""
```

Comment:
This made this implementation untestable and untested. Whenever you require the file it would run through this code.

Reply:
Thanks for catching that.